MAINTAINERS_GUIDE.md

```diff
@@ -66,25 +66,6 @@ by anyone is denoted by adding a comment in the pull request: `LGTM`.
 However, only currently listed `MAINTAINERS` are counted towards the required
 two LGTMs.
 
-Overall the maintainer system works because of mutual respect across the
-maintainers of the project.  The maintainers trust one another to make decisions
-in the best interests of the project.  Sometimes maintainers can disagree and
-this is part of a healthy project to represent the point of views of various people.
```

Comment:
Perhaps up until line 72 is still ok to keep?

Reply:
done